tests/test_citeprocpy.py

```diff
@@ -1,62 +1,10 @@
 # -*- coding: utf-8 -*-
-import os
-import json
-
 from django.utils import timezone
 from nose.tools import *  # noqa: F403
 
 from api.citations.utils import render_citation
 from osf_tests.factories import UserFactory, PreprintFactory
 from tests.base import OsfTestCase
-from osf.models import OSFUser
-
-class Node:
-    _id = '2nthu'
-    csl = {'publisher': 'Open Science Framework', 'author': [{'given': u'Henrique', 'family': u'Harman'}],
-           'URL': 'localhost:5000/2nthu', 'issued': {'date-parts': [[2016, 12, 6]]},
-           'title': u'The study of chocolate in its many forms', 'type': 'webpage', 'id': u'2nthu'}
-    visible_contributors = ''
-
-
-class TestCiteprocpy(OsfTestCase):
```

Comment:
I understand that the citation tests `test_failing_citations` and `test_passing_citations` require a couple of migrations that we moved out of the stream and becomes a Django command early in this PR. Is there a plan to add the tests back later after the upgrade where we can populate test database via fixtures? If so, I suggest skip these two tests and add some TODO comments.

Reply:
Jon and I looked at these tests and we were a little confounded by what these tests were actually doing. These tests check against a pre-generated list of thousands of citations, but we don't actually know if that list is accurate. The test fixture is quite big too and that factored into our decision to just remove it.